.env.example

```diff
@@ -1 +0,0 @@
-DATABASE_URL="postgres://user:password@localhost:5432/DB_DEVELOPMENT_DOCKER"
```

Comment:
why did you remove this file?

Reply:
I didn't remove the file, instead, I have just renamed the file by removing the [.example] from the file name as instructed in the README file. However the default Postgres credentials were not working on my end until I used my system Postgres credentials.

With the default credential set in docker, I was getting this error,
**Error: P1010: User user was denied access to the database DB_DEVELOPMENT_DOCKER.public**

Reply:
got it. this PR would remove the `.env.example` from the main branch. can you remove this change? i.e. add the example env files again

> I didn't remove the file, instead, I have just renamed the file by removing the [.example] from the file name as instructed in the README file.

will update the README to 👍

Reply:
Yeah, let me add the file back

Reply:
updated the readme

Reply:
Okay thanks